forge-gui/res/adventure/Shandalar/custom_cards/scarecrow_captain.txt

```diff
@@ -1,11 +1,10 @@
 Name:Scarecrow Leader
 ManaCost:no cost
 Types:Enchantment
```

Comment:
card says enchantment, image says artifact.. was gonna create a new card image but what is the final type?

Reply:
Generally I have been using enchantment for boss effects but Artifact could be fitting as well (since Scarecrow in MTG are artifacts)